.gitignore

```diff
@@ -22,3 +22,4 @@ cache
 
 # env
 .envrc
+/out/
```

Comment:
need this?

Reply:
For JetBrains-based IDEs, it auto-generates a "compiled" output -- I don't think it hurts to have it